stories/components/MenuItem/MenuItem.tsx

```diff
@@ -1,11 +1,16 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
Xóa cái này đi. `ReactNode` có làm gì đâu